fuel_test/prep_env.sh

```diff
@@ -186,7 +186,9 @@ FM_IP=$(wait_for_fm "$XS_HOST" "$FM_NAME" 60 10)
 NAILGUN_READY=$(wait_for_nailgun "$FM_IP" 60 10)
 [ $NAILGUN_READY -eq 0 ] && echo "Nailgun test connection timeout" && exit -1
 
+sleep 5
```

Comment:
Previously you mentioned if we add some sleeps before/after the vm's boot maybe it will be less easily to get stuck.